.github/workflows/all.yml

```diff
@@ -83,6 +83,28 @@ jobs:
           fi
           go test "${args[@]}" -timeout 800s ./...
 
+  tc39:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+      - name: Install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.15.x
+      - name: Run tests
+        run: |
+          set -x
+          cd js/tc39
+          mkdir -p TestTC39/test262
+          cd TestTC39/test262
+          git init
+          git remote add origin https://github.com/tc39/test262.git
+          git fetch origin --depth=1 72154b17fc99a26e79b2586960f059360d4ce43d
```

Comment:
What is so special about this commit? And if you don't want to use `master` because of potential future flakyness, can we have this commit ID as an environment variable, so it'd be a bit more visible and easier to update.

Reply:
This was the latest commit at the time I started :D